data/deprecated.json

```diff
@@ -1651,6 +1651,18 @@
     "old": {"townhall:village": "yes"},
     "replace": {"amenity": "townhall", "townhall:type": "village"}
   },
+  {
+    "old": {"transformer": "auto"},
+    "replace": {"windings:auto": "yes"}
```

Comment:
to me it looks like as if a `windings:auto=yes` tag alone without any value specified for `transformer=*` doesn't make much sense. I don't know if the previous `transformer=auto` tag was only intended for substation transformers which are now to be tagged as `transformer=main`, or if it could have been any kind of autotransformer (e.g. auxiliary, distribution, etc.). If it is the first option, then the `replace` line should have also the `transformer=main` tag included, but if it is the second option, it does not make sense to add this as a deprecation rule here:

A mapper would need to do research or perform a survey to fix these values. Such a task could be better coordinated through a maproulette project for example.

Reply:
It's indeed necessary to look for the appropriate transformer value (it could be main, auxiliary, converter...)
I can update this deprecation with transformer=yes, is it acceptable?

Reply:
>  I can update this deprecation with transformer=yes, is it acceptable?

Yes, please!